docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,1748 @@
+       no direct replacement.
+
+   * - :class:`~qiskit.opflow.primitive_ops.CircuitOp`
+     - :class:`~qiskit.circuit.QuantumCircuit`
+
+   * - :class:`~qiskit.opflow.primitive_ops.MatrixOp`
+     - :class:`~qiskit.quantum_info.Operator`
+
+   * - :class:`~qiskit.opflow.primitive_ops.PauliOp`
+     - :class:`~qiskit.quantum_info.Pauli`. For direct compatibility with classes in :mod:`qiskit.algorithms`,
+       wrap in :class:`~qiskit.quantum_info.SparsePauliOp`.
+
+   * - :class:`~qiskit.opflow.primitive_ops.PauliSumOp`
+     - :class:`~qiskit.quantum_info.SparsePauliOp`. See example below.
+
+   * - :class:`~qiskit.opflow.primitive_ops.TaperedPauliSumOp`
+     - This class was used to combine a :class:`.PauliSumOp` with its identified symmetries in one object.
+       This functionality is not currently used in any workflow, and has been deprecated without replacement.
+       See |qiskit.quantum_info.Z2Symmetries|_ example for updated workflow.
+
+   * - :class:`qiskit.opflow.primitive_ops.Z2Symmetries`
+     - |qiskit.quantum_info.Z2Symmetries|_. See example below.
+
+.. _pauli_sum_op:
+
+.. raw:: html
+
+    <details>
+    <summary><a><font size="+1">Example 1: <code>PauliSumOp</code></font></a></summary>
+    <br>
+
+**Opflow**
+
+.. code-block:: python
+
+    >>> from qiskit.opflow import PauliSumOp
+    >>> from qiskit.quantum_info import SparsePauliOp, Pauli
+
+    >>> qubit_op = PauliSumOp(SparsePauliOp(Pauli("XYZY"), coeffs=[2]), coeff=-3j)
+    >>> qubit_op
+    PauliSumOp(SparsePauliOp(['XYZY'],
+                  coeffs=[2.+0.j]), coeff=(-0-3j))
+
+**Alternative**
+
+.. code-block:: python
+
+    >>> from qiskit.quantum_info import SparsePauliOp, Pauli
+
+    >>> qubit_op = SparsePauliOp(Pauli("XYZY"), coeffs=[-6j])
+    >>> qubit_op
+    SparsePauliOp(['XYZY'],
+                  coeffs=[0.-6.j])
+
+.. raw:: html
+
+   </details>
+
+.. _z2_sym:
+
+.. raw:: html
+
+    <details>
+    <summary><a><font size="+1">Example 2: <code>Z2Symmetries</code> and <code>TaperedPauliSumOp</code></font></a></summary>
+    <br>
+
+**Opflow**
+
+.. code-block:: python
+
+    >>> from qiskit.opflow import PauliSumOp, Z2Symmetries, TaperedPauliSumOp
+
+    >>> qubit_op = PauliSumOp.from_list(
+    ...    [
+    ...    ("II", -1.0537076071291125),
+    ...    ("IZ", 0.393983679438514),
+    ...    ("ZI", -0.39398367943851387),
+    ...    ("ZZ", -0.01123658523318205),
+    ...    ("XX", 0.1812888082114961),
+    ...    ]
+    ... )
+    >>> z2_symmetries = Z2Symmetries.find_Z2_symmetries(qubit_op)
+    >>> tapered_op = z2_symmetries.taper(qubit_op)
+
+    # can be represented as:
+    >>> tapered_op = TaperedPauliSumOp(qubit_op.primitive, z2_symmetries)
+
+    >>> print(z2_symmetries)
+    Z2 symmetries:
+    Symmetries:
+    ZZ
+    Single-Qubit Pauli X:
+    IX
+    Cliffords:
+    0.7071067811865475 * ZZ
+    + 0.7071067811865475 * IX
+    Qubit index:
+    [0]
+    Tapering values:
+      - Possible values: [1], [-1]
+
+    >>> tapered_op
```

Comment:
Above tapered_op is redefined. What were you wanting to print here? This is the original operator - versus below where its a list of the operators according to the taperings.